docs/analytics-ga4/ga4-finder-tracker.md

```diff
@@ -71,10 +71,13 @@ If the filter `<select>` element changes and is set to their _first option_, we
 
 Radio buttons have `data-ga4-change-category="update-filter radio"` on them. If the changed radio button is not the _first radio button_ (i.e. the default selection), then, the `update-filter` event will build an "Filter added" GTM Object. If the changed radio button is the _first radio button_, it will build a "Filter removed" GTM Object. This is because the first radio button in a list is typically the default "All XYZ" filter state. The value we grab for a radio button is the user friendly label associated with the `<input>`.
 
-### Date filters (`<input type="text">`)
+### Date filters with single input field (`<input type="text">`)
```

Comment:
I've no qualms with you using "Legacy" on the description of these if that is helpful in explaining them - they'll always be legacy from this point until they're removed.

Reply:
Decided to forgo "legacy" as we are still using them in some contexts and we don't know if/when we'll get rid of them!